platzky/models.py

```
@@ -21,28 +21,70 @@ class CmsModuleGroup(CmsModule):
     author: str
     comment: str
     date: str  # TODO change its type to datetime
 
     @property
     def time_delta(self) -> str:
+        """Calculate human-readable time since the comment was posted.
+
+        Returns:
+            Human-friendly time description (e.g., "2 hours ago", "3 days ago")
+        """
         now = datetime.datetime.now()
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)